packages/sdk/src/provider/extensionProviderHelpers/handleUuid.ts

```diff
@@ -0,0 +1,25 @@
+import { v4 as uuidv4 } from 'uuid';
+import { base64Encode } from '../../utils/base64';
+/**
+ * Gets or creates a unique identifier (UUID) based on the provided url and name.
+ * The identifier is stored in localStorage using a Base64 encoded combination of `url` and `name`.
+ *
+ * @param {string} url - The URL of the dapp.
+ * @param {string} name - The name of the dapp.
+ * @returns {string} - The unique identifier (UUID) for the dapp.
+ */
+function getOrCreateUuidForIdentifier(url: string, name: string): string {
```

Comment:
I think we should use object parameters as much as possible everywhere.
It adds benefit to easily identify what is being passed to the function and also makes it very easy to change the method later while staying backward compatible.
It is not everywhere in the codebase because it wasnt started this way but makes things easier to maintain.

Reply:
done